packages/xo-web/src/xo-app/pool/tab-advanced.js

```diff
@@ -396,6 +400,20 @@ export default class TabAdvanced extends Component {
                       <Toggle value={pool.auto_poweron} onChange={this._onChangeAutoPoweron} />
                     </td>
                   </tr>
+                  <tr>
+                    <th>{_('migrationCompression')}</th>
+                    <td>
+                      <Tooltip
+                        content={isMigrationCompressionAvailable ? undefined : _('migrationCompressionDisabled')}
+                      >
+                        <Toggle
+                          value={pool.migrationCompression}
+                          onChange={this._onChangeMigrationCompression}
+                          disabled={isMigrationCompressionAvailable}
```

Comment:
This one is wrong too: [see my comment](https://github.com/vatesfr/xen-orchestra/pull/7642#discussion_r1618553813)
```suggestion
                          disabled={!isMigrationCompressionAvailable}
```